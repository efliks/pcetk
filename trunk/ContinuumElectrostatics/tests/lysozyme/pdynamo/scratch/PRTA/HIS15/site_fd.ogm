ON_GEOM_CENT 121 2.000000
(-9.686999 12.798260 17.254403) 101 1.000000
(-9.686999 12.798260 17.254403) 101 0.500000
(-9.686999 12.798260 17.254403) 101 0.250000
