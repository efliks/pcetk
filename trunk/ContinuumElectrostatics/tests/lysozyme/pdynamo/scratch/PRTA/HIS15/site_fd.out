Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/HIS15//site_fd
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.664701, 20.5141, 19.0512)) 121 2
Finer level:    (-9.687, 12.7983, 17.2544) 101 1
Finer level:    (-9.687, 12.7983, 17.2544) 101 0.5
Finest level:   (-9.687, 12.7983, 17.2544) 101 0.25


Self energy of scratch/PRTA/HIS15//site_fd in three dielectric environment = 278.774
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/HIS15//site_fd with background charge set in three dielectric environment = -0.161511
(probably in kcal/mole)



Interaction energies of scratch/PRTA/HIS15//site_fd with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.357235
0   1 : 0.00675969
1   0 : -0.0244288
1   1 : 0.263277
2   0 : -0.135414
2   1 : -0.0021333
3   0 : 378.2
3   1 : 407.807
3   2 : 419.533
3   3 : 557.552
4   0 : -0.0341339
4   1 : 0.0912881
5   0 : 0.00281265
5   1 : 0.130754
6   0 : 0.00460235
6   1 : 0.0344039
7   0 : -0.0254404
7   1 : -0.00058115
8   0 : -0.0130272
8   1 : 0.0561424
9   0 : 0.000710624
9   1 : 0.00821771
10   0 : -0.00480556
10   1 : 0.0307059
11   0 : 0.000230002
11   1 : 0.0385343
12   0 : -0.00249993
12   1 : 0.0223743
13   0 : 0.131749
13   1 : 1.22239
14   0 : -0.457401
14   1 : 0.00121763
15   0 : -0.0709018
15   1 : -0.000613585
16   0 : 0.00261764
16   1 : 0.0375882
17   0 : -0.00803756
17   1 : -5.85471e-05
18   0 : 0.000127604
18   1 : 0.00972617
Total runtime: 16.62 s user 0.13 s system 
